src/latlon.py

```diff
@@ -12,14 +12,18 @@ def getlatlon(imgpath):
     im = rasterio.open(imgpath)
     crs = im.crs.__str__()  # Coordinate reference system code
     nrows, ncols = im.shape
-    rows, cols = np.meshgrid(np.arange(nrows), np.arange(ncols))
+    cols, rows = np.meshgrid(np.arange(ncols), np.arange(nrows))
     xs, ys = rasterio.transform.xy(im.transform, rows, cols)
+    xs = np.array(xs)
+    ys = np.array(ys)
+    
     # X and Y are the 1D index vectors
-    X = np.array(xs)[:, 0]
-    Y = np.array(ys)[0, :]
+    X = np.array(xs)[0, :]
+    Y = np.array(ys)[:, 0]
     ps_to_ll = Transformer.from_crs(im.crs, "WGS84", always_xy=True)
     lons, lats = ps_to_ll.transform(np.ravel(xs), np.ravel(ys))
+    
     # longitude and latitude are 2D index arrays
     longitude = np.reshape(lons, (nrows, ncols))
     latitude = np.reshape(lats, (nrows, ncols))
-    return {"crs": crs, "longitude": longitude, "latitude": latitude, "X": X, "Y": Y}
+    return {"crs": crs, "longitude": longitude, "latitude": latitude, "X": X, "Y": Y}
```

Comment:
This line is the same too but GitHub is picky about not leaving an empty line and the end of the file.

Reply:
Funny how it's picky about that here, but when I delete the empty line and save the file, it's not enough to register it as a change when I check "git status"